src/components/prompt/prompt.jsx

```diff
@@ -46,6 +46,8 @@ const PromptComponent = props => (
             <Box>
                 <input
                     autoFocus
+                    autoCapitalize={'none'}
```

Comment:
With JSX you can just use `autoCapitalize="none"` instead of the curly brackets + single quotes. Could you change that in this file and the others?

Reply:
I updated this : )